### ISSUE TITLE
Implement  AIC and BIC

### ISSUE THREAD

**Issue by adam-m-mcelhinney** [mfrasco/Metrics#21]
I propose that the AIC and BIC functions are implemented. This would be easy to do on top of the logloss that already exists. 
https://en.wikipedia.org/wiki/Akaike_information_criterion
https://en.wikipedia.org/wiki/Bayesian_information_criterion

Note that other implements of AIC are not guaranteed to give the actual AIC value, only an approximation that is missing a constant. I am not sure if that is because of the loglikelihood function or bc of the AIC function.

**Comment by mfrasco** [comment]
Yes, this is a great metric. We should specify an argument that indicates the distribution type. Also, we should try to make it so that our implementation matches `stats::AIC`. So we might need to figure out what that constant is.

**Comment by timbook** [comment]
Is this still open/desired? I'd like to tackle this. Is there also any interest in Mallow's Cp?
